pages/common/rustfmt.md

```diff
@@ -0,0 +1,15 @@
+# rustfmt
+
+> Tool for formatting Rust source code.
+
+- Format a file, overwriting the original file in-place:
+
+`rustfmt {{source.rs}}`
+
+- Check a file for formating and display any changes on console:
+
+`rustfmt --check {{source.rs}}`
+
+- Backup any modified files before formatting. If a file source.rs is formated, the backup is source.bk:
```

Comment:
Typo: `formated` -> `formatted`.

Perhaps `source.rs` and `source.bk` should be enclosed in backticks?

Also, the full-stop here is a bit weird. perhaps we could put the second sentence in brackets?

Reply:
sure